apps/site/src/components/pages/pricing/paid-addons-section.tsx

```diff
@@ -0,0 +1,658 @@
+        >
+          <strong>Paid</strong>{" "}
+          <Box
+            component="span"
+            sx={{ color: ({ palette }) => palette.gray[60] }}
+          >
+            ADD-ONS
+          </Box>
+        </Typography>
+
+        <Grid
+          container
+          sx={{
+            display: "grid",
+            gridTemplateAreas: {
+              xs: `'usage' 'dev' 'support'`,
+              lg: `'usage dev'
+            'usage support'`,
+            },
+            gridGap: 20,
+          }}
+        >
+          <Section sectionName="usage" fullWidth>
+            <Stack flexDirection="row" alignItems="center" mb={2.25}>
+              <GradientFontAwesomeIcon
+                icon={faCode}
+                sx={{ mr: 1.5, height: 32, width: 42 }}
+                light
+              />
+
+              <Typography
+                variant="bpHeading3"
+                sx={{
+                  lineHeight: 1.4,
+                  color: ({ palette }) => palette.gray[90],
+                }}
+              >
+                Additional Usage
+              </Typography>
+            </Stack>
+
+            <Stack gap={2.25}>
+              <Typography variant="bpBodyCopy" sx={{ lineHeight: 1.4 }}>
+                <strong>
+                  API usage beyond your included allowance will be billed in
+                  line with the below.
+                </strong>{" "}
+                Hobby and Pro users can prevent overage charges by setting a{" "}
+                <CustomLink href="/account/billing">spend cap</CustomLink> from
+                the account billing page.
+              </Typography>
+
+              <Box sx={{ overflowX: "auto" }}>
+                <Table
+                  header={API_USAGE_HEADER.map((title) => (
+                    <UsageTableHeaderCell key={title}>
+                      {title}
+                    </UsageTableHeaderCell>
+                  ))}
+                  rows={API_USAGE_ROWS.map((row) => createTableRow(row))}
+                />
+              </Box>
+
+              <Typography variant="bpBodyCopy" sx={{ lineHeight: 1.4 }}>
+                <strong>
+                  We strive to keep these prices in line with the ordinary cost
+                  of these services.
+                </strong>{" "}
+                If you believe any of these prices are out of sync with the
+                costs charged by the underlying provider,{" "}
+                <CustomLink href="/contact">please let us know</CustomLink>. A
+                Platform Service Fee of <i>30%</i> for Free users, <i>25%</i>{" "}
+                for Hobby users, and <i>20%</i> for Pro users, is applied to
+                Additional API Usage calls to cover costs.
+              </Typography>
+
+              <Typography variant="bpBodyCopy" sx={{ lineHeight: 1.4 }}>
+                <strong>Want to list an endpoint?</strong>{" "}
+                <CustomLink
+                  href="/docs/services"
```

Comment:
I have a ticket to add this page, which we said I'd do last-minute given your other docs changes, to avoid conflicts ([internal link](https://app.asana.com/0/1203623179643284/1203845851310055/f))

Reply:
i.e. let's leave this link as-is, but remember to complete the sprint objective ^